exec/java-exec/src/main/java/org/apache/drill/exec/record/metadata/TupleSchema.java

```diff
@@ -179,18 +184,16 @@ public String fullName() {
 
   @Override
   public String toString() {
-    StringBuilder buf = new StringBuilder()
+    StringBuilder builder = new StringBuilder()
         .append("[")
         .append(getClass().getSimpleName())
         .append(" ");
-    boolean first = true;
-    for (ColumnMetadata md : nameSpace) {
-      if (! first) {
-        buf.append(", ");
-      }
-      buf.append(md.toString());
-    }
-    buf.append("]");
-    return buf.toString();
+
+    builder.append(nameSpace.entries().stream()
+      .map(ColumnMetadata::toString)
+      .collect(Collectors.joining(", ")));
```

Comment:
Show off! That's some fancy functional code for such a simple task... Much cleaner.